Documentation/DemoApps/GRDBCombineDemo/GRDBCombineDemoUITests/GRDBCombineDemoUITests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GRDBCombineDemoUITests.swift
+//  GRDBCombineDemoUITests
+//
+//  Created by Peter Steinberger on 08.04.21.
+//
+
+import XCTest
+
+class GRDBCombineDemoUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+    }
+
+    override func tearDownWithError() throws {
+    }
+
+    func runApp() -> XCUIApplication {
+        // UI tests must launch the application that they test.
+        let app = XCUIApplication()
+        // We enforce a fresh database to ensure 8 random players.
+        app.launchArguments = ["-reset", "-fixedTestData"]
+        app.launch()
+        return app
+    }
+
+    func testInitialSortingIsByScore() throws {
+        let app = runApp()
+
+        let getFirstCell = { () -> XCUIElement in
+            app.tables.element(boundBy: 0).cells.element(boundBy: 0)
+        }
+
+        // Given default sorting, Henriette must be on top
+        XCTAssertEqual(getFirstCell().label, "Craig, 8 points")
```

Comment:
This test fails here without my query change.

Reply:
Thank you so much, so much, for this. 

Now... I have ruined your efforts in https://github.com/groue/GRDB.swift/pull/955/commits/a5ffc52dbec8c6277ef1de70c0e1e7903f4c339a, in order to make the demo app... a demo app, and not a test case. This test is still there, but it no longer checks for your setup, because I have removed the conditions where your `@Query` bugfix expresses itself 😅 

But I hope I have clearly expressed what we are talking about in https://github.com/groue/GRDB.swift/pull/955/commits/c34615ced323aa5802a7b61afdb997b93f71e1a4. I quite intend to support your use case in the future, and setup stable tests.

Until then, this does not prevent this pull request from being merged.